src/i18n/es.json

```diff
@@ -8,5 +8,12 @@
     "iso": "Certificación de Calidad ISO 9001:2008",
     "privacy": "Políticas de privacidad y legales.",
     "privacy_url": "https://fromdoppler.com/privacidad"
+  },
+  "upgradePlanForm": {
+    "title": "Solicita una actualización de tu plan",
+    "plan_select": "Selecciona el Plan",
+    "message": "Mensaje",
+    "send": "Enviar",
+    "cancel": "Cancelar"
```

Comment:
What do you think about making these literals global? I think that we will reuse them a lot.

Reply:
in the same file? or create something like common.es and common.en ?

Reply:
Creo que en el mismo, pero fuera de la sección "upgradePlanForm" o en una sección "general" o "common", que opinás?